### ISSUE TITLE
Unable to Move Pieces!

### ISSUE THREAD

**Issue by artem-kuzishchin** [lichess-org/lichobile#2056]
**Describe the bug**
Very often (three times today alone), the mobile app will not let me select and move a piece when it is my turn in-game. My opponent will move, and I will attempt to reply, but no piece on the board can be selected, and no moves can be made. It feels pretty bad to lose on time because the application literally does not let me play a move. It can occasionally be remedied by pressing the "back-a-move" button, followed by the "forward-by-a-move" button to get back to the active game state, but this fix does not always work.

**To Reproduce**
This is erratic- it doesn't seem to happen every game, but it happens often.

**Expected behavior**
Being able to play the game lmao

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy S9
 - OS: Android 10

**Additional context**

**Comment by veloce** [comment]
To be sure to understand: only pieces don't move, but all other buttons are working (like game controls, menu, etc.), right?

**Comment by artem-kuzishchin** [comment]
Yeah, every other part of the app works fine, it's just selecting and moving pieces- I can offer a draw and explain the situation through chat, but no matter what the pieces just won't move. This has also happened in puzzles and on the analysis board, so it's not just specific to online games. I've found restarting the app also works, but I'm hesitant to do it in a live game.

**Comment by mcstatu** [comment]
It happens exactly the same, I could not select and move any piece on the board, I lost over time. And I even received a message from server complaining about letting time run instead of playing... "It was not my fault, it's a nasty bug in the android app"

**Comment by surfstian** [comment]
Similar issues described on #1951

**Comment by TestPlan** [comment]
I have the same problem on the Android app. I notice this happens after using the chat. If I avoid chat, games typically play out smoothly. If I say anything in chat, then at some point during the game, the pieces become unmovable

**Comment by mcstatu** [comment]
And seams that no one wants to do nothing to correct the bug, more than 2 years experiencing this problem...

**Comment by csaavedra** [comment]
This bug is really annoying, specially in when playing bullet. The only way to get to move pieces again is by going back and forth with the arrows, but in bullet that basically means losing the game. How many more years before this is actually taken care of?

**Comment by jas14** [comment]
Hello all, this repository is deprecated. There is a complete rewrite in Flutter underway at lichess-org/mobile; that has almost all the same functionality (and in fact some new features, too) as this version. You may opt into beta builds at https://lichess.org/forum/team-lichess-beta-testers/mobile-app-alpha?page=1 (or, of course, you can always build and install from source).

Hope this helps!

**Comment by csaavedra** [comment]
Hm, I understand that resources are scarce, but you are basically saying that the stable app is unmaintained and that people should just use an app that is still in beta stage (and probably has bugs). Neither option sounds convincing to be honest.

If the repo/app is really deprecated you should probably blacklist it from connecting to the server and remove it from the app stores, otherwise users are left with dealing with these bugs, the rating loses they can cause, etc. Not a very responsible approach.

**Comment by jas14** [comment]
@csaavedra these are valid points and I hear what you're saying. If it makes it any better, you're certainly [not alone](https://discord.com/channels/280713822073913354/807722604478988348/993963850938450000) in feeling this way.

I'm not the maintainer of this codebase or part of Lichess organizational decision-making, but if I had to hazard a guess, I'd bet the reasoning here was that the technology underpinning this repo (Capacitor) was fundamentally broken, or at least so difficult to work with that it warranted a complete rewrite. Furthermore, as you noted, resources are scarce. The product of both of these is many unsolved issues, like the very one we're commenting on now.

I understand and agree that being forced to choose between a known buggy app, a probably buggy app, and a mobile site is not pleasant. I'm hopeful that the Flutter app will be publicly released this year. In any case, I hope you give it a try before then - I've been using it myself and am very pleased with its features, responsiveness, and correctness.

Please feel free to join the discussion on Lichess Discord too - #lichess-mobile-app in particular.
